2024/06.md

```diff
@@ -120,6 +120,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     | timebox | topic | presenter |
     |:-------:|-------|-----------|
     | 60m     | Source maps progress update | Jonathan Kuperman and Agata Belkius |
+    | 60m     | Smart units progress update | Ben Allen |
```

Comment:
what is this 👀

Reply:
I have the same question - is there a URL for this? I'm not familiar with "smart units".

Reply:
ping @ben-allen

Reply:
probably this (?)  https://github.com/tc39/proposal-smart-unit-preferences

Reply:
here's the notes from last time to refresh your memory: https://github.com/tc39/notes/blob/840c700dc7fa7b9f6d0a3c208bd66b333e304717/meetings/2020-06/june-4.md#smart-unit-preferences-in-intlnumberformat-for-stage-1

Reply:
ping @younies; is Ben taking over this proposal?

Reply:
Yes, I'm taking this one over. Apologies for the mysterious agenda addition &mdash; supporting materials TK shortly